src/core/BUILD

```diff
@@ -1108,6 +1109,7 @@ grpc_cc_library(
         "absl/status",
         "absl/strings:str_format",
     ],
+    visibility = ["@grpc:alt_grpc_base_legacy"],
```

Comment:
Just curious about the meaning of this?

Reply:
Means that things that can see grpc_base can see this target too... which for now is approximately what we want.